docs/rewards_fees.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Rewards & Fees
+sidebar_position: 8
+description: Rewards and Fees paid and received by Subspace Network participants
+keywords:
+    - Rewards
+    - Fees
+    - Tokenomics
+---
+
+All the participants of the Subspace Network are compensated for the work they do to keep the network live and secure. In this context, we define the following terms:
+- **Fees**: The payments for transactions on the Subspace Network.
+- **Rewards**: The compensation for the work done by the participants of the Subspace Network.
+- **Issuance**: The issuance of the newly minted SSC by the protocol.
+
+Different participants receive their compensation through a combination of the above based on their role.
+
+## Farmers
+
+Currently, farmer rewards include: 
+- fees for the transactions they include in consensus chain blocks,
+- block rewards for the blocks they proposed, issued by the protocol,
+- vote rewards issued by the protocol,
+- Designated Storage Fund (DSF) subsidy.
+
+## Operators 
+
+Currently, operators are solely rewarded with the fees for domain transactions they execute. Since domain transactions (e.g., EVM contract calls) are usually much more computationally heavy, than those performed on the consensus chain (e.g., balance transfers), they are substantially expensive to compensate operators fairly for their work. The operators receive the reward for the executed transactions only after the corresponding domain block has cleared the challenge period. For more details, see [Domain Block Rewards](/docs/decex/domains/workflow.md#domain-block-rewards).
```

Comment:
```suggestion
Currently, operators are solely rewarded with the fees for domain transactions they execute. Since domain transactions (e.g. EVM contract calls) are usually much more computationally heavy, than those performed on the consensus chain (e.g. balance transfers), they are substantially expensive to compensate operators fairly for their work. The operators receive the reward for the executed transactions only after the corresponding domain block has cleared the challenge period. For more details, see [Domain Block Rewards](/docs/decex/domains/workflow.md#domain-block-rewards).
```